### DISCUSSION TITLE
Any way to open the terminal in non-administrator mode?

### DISCUSSION THREAD

**Discussion by ySergi** [microsoft/terminal#17254] [Q&A]
I have by default the terminal to open in administrator mode, I know it can be disabled again in the options, but what I want to know is if there is a way to open the terminal in normal mode even if it is set to open in administrator mode by default.

**Comment by zadjii-msft** [comment]
As of 1.21, not really. I'm pretty sure we looked into de-elevating specific profiles, and IIRC for packaged apps, that's not really possible once they're already elevated.
